### PR TITLE
Check for pdo_mysql extension, mysqli is no longer used

### PR DESCRIPTION
**Pull request by Derkades**
With #3198, the last usages of mysqli will be removed. Now that `mysqli` is no longer required by NamelessMC, check for presence of the `pdo_mysql` extension instead.

### CODE REVIEW

**Review by tadhgboyle — COMMENTED** [review]
Is `pdo_mysql` going to be as common as `mysqli`?
As in, does this make it even harder to install for people on bad hosts?

**Comment by Derkades** [comment]
`pdo_mysql` is an adapter for PDO, `mysqli` is something else entirely. NamelessMC uses `pdo_mysql`, but never checked if it was installed. This is not really an issue, virtually every host that has `PDO` installed also has the `pdo_mysql` extension installed.

`mysqli` is no longer used by NamelessMC so the check for it can be removed.

TL;DR: This does not worsen compatibility, instead it improves compatibility by not requiring extensions that are not used

**Review by tadhgboyle — APPROVED** [review]
